vita3k/config/include/config/config.h

```diff
@@ -86,14 +86,16 @@
     code(bool, "show-welcome", true, show_welcome)                                                      \
     code(bool, "asia-font-support", false, asia_font_support)                                           \
     code(bool, "video-playing", true, video_playing)                                                    \
-    code(bool, "spirv-shader", false, spirv_shader)
+    code(bool, "spirv-shader", false, spirv_shader)                                                     \
+    code(uint64_t, "current-ime-lang", 4, current_ime_lang)
```

Comment:
Is uint64_t really necessary, maybe uint32_t

Reply:
number can be get very big i have used value set for supportedlanguage inside param, is uint64_t, and is this value can give it of lang